src/index.js

```diff
@@ -21,3 +22,6 @@ desc.addEventListener('keypress', (e) => {
 const load = () => window.location.reload();
 
 refresh.addEventListener('click', load);
+clear.addEventListener('click', () => {
+  todo1.clearCompleted();
+});
```

Comment:
- [ ] The tasks state is not correctly saved in the LocalStorage. Checked tasks are not turned to `completed` in your state. This has the consequence that after a page reloads any previously checked task is not checked anymore. Let's fix that

<img width="1171" alt="Screen Shot 2022-08-18 at 11 07 33 AM" src="https://user-images.githubusercontent.com/40503704/185370649-6825984a-d995-48c5-9f3a-5c23fcfc875c.png">